src/app/services/user-service.ts

```diff
@@ -1,31 +1,36 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { UserEditComponent } from '../user-edit/user-edit.component';
-
+import { Observable } from 'rxjs';
 
 @Injectable()
+
+
 export class UserService {
   constructor(private http: HttpClient) { }
 
-  getJammers() {
-    return this.http.get('https://1d9a3efd-bdd5-4feb-baad-39d400f80776.mock.pstmn.io/api/v1/user/1/search?instrument=guitar');
-  }
+  // getJammers() {
+  //   return this.http.get('https://1d9a3efd-bdd5-4feb-baad-39d400f80776.mock.pstmn.io/api/v1/user/1/search?instrument=guitar');
+  // }
 
   getUserProfile() {
-    return this.http.get('https://1d9a3efd-bdd5-4feb-baad-39d400f80776.mock.pstmn.io/api/v1/user/1');
+    return this.http.get('http://letusjam.herokuapp.com/api/v1/users/1');
```

Comment:
Could this URL be dynamic? Or could all interactions of the API exist in a util file?

Reply:
Since the project requirements discouraged us from using auth or logins, we decided to hard code the user that is currently logged in. If we were to add a login feature, making this dynamic is absolutely one of the first things we would change. All of the interactions with the API currently exist in the user services component so that they're accessible all around the app. With my limited knowledge of Angular and our short remaining time limit, not exactly sure how moving them to a util would work.